src/transformers/searchservice/HighlightedValueFactory.ts

```diff
@@ -34,24 +34,22 @@ export class HighlightedValueFactory {
     if (typeof data !== 'object' || data === null){
       return [];
     }
+    if (this.isChildHighlightedField(data)) {
+      return [ this.from(data.value, path[path.length - 1], path, data.matchedSubstrings) ];
```

Comment:
nit: adding a `const fieldName = path[path.length - 1];` and passing that in as the second parameter might make this a little easier to understand

Reply:
yeah I like it! I was actually rereading my code and had to do a double take on why I did this lol